generators/client/templates/angular/src/main/webapp/app/admin/audits/_audits.component.ts

```diff
@@ -4,6 +4,7 @@ import { DatePipe } from '@angular/common';
 import { Audit } from './audit.model';
 import { AuditsService } from './audits.service';
 import { ParseLinks } from '../../shared';
+import { ITEMS_PER_PAGE } from '../../shared/constants/pagination.constants';
```

Comment:
please import this in the above import as `{ ParseLinks, ITEMS_PER_PAGE } from '../../shared'`